src/ik_test.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016-2017, Philipp Sebastian Ruppel
```

Comment:
Where did this file come form?

Reply:
same reason for ik_evolution_1.h in above comment -- so we can call `makeTestSolver` elsewhere.

This comes back to whether or not enhancements should be included in this PR--the way we instantiate the solvers here isn't how the original code does things